src/tiff.ml

```diff
@@ -821,14 +821,22 @@ module Ifd = struct
     { entries; data_offsets; data_bytecounts; ro = reader; header }
 end
 
-type t = { header : header; ifd : Ifd.t }
+(* I imagine one day we will also have `Some of int` as an option here *)
```

Comment:
What will that do? Only cache `n` strips?

I do wonder if we should not make the `ro` more sophisticated here so the TIFF library doesn't have to deal with this. A user could already do something very similar themselves using the file_offset as the cache key?